core/templates/dev/head/pages/email_dashboard/email_dashboard_result.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'pages/base.html' %}
+
+{% block maintitle %}
+  Email Dashboard Result - Oppia
+{% endblock maintitle %}
+
+{% block header_js %}
+  {{ super() }}
+  <style>
+    html, body {
+      background-color: #f2f2ee;
+    }
+    .success {
+      color: #0000ff;
+    }
+    .oppia-email-form {
+      text-align: center;
+      margin-top: 2%;
+    }
+    textarea {
+      margin-bottom: 1%;
+      border-radius: 10px;
+    }
+    .invalid-input {
+      color: red;
+    }
+    .success {
+      color: green;
+    }
+  </style>
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      Email Dashboard
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div class="oppia-email-form" ng-controller="EmailDashboardResult">
+    <p ng-if="emailSubmitted" class="success">Emails sent successfully. You will be redirected to Email Dashboard page.</p>
```

Comment:
You will be redirected to Email Dashboard page. --> Redirecting to the Email Dashboard...